### PR TITLE
Add URL encoding for multiple GET parameters

### PR DESCRIPTION
**Pull request by dim0x69**
Disclaimer: I only looked at Jira until now :)

Currently there seems not to be a method which adds multiple GET parameters to the API request. Indeed, these might happen in the future (I will add another Pull Request  to add the "expand" keyword to Jira.issue_field_value() and Jira.issue() ;) )

This Pull Request adds the option to have multiple GET parameters in the URL.

### CODE REVIEW

File: atlassian/rest_client.py [hunk 1]
```diff
@@ -197,6 +197,8 @@ def request(
         params_already_in_url = True if "?" in url else False
         if (params or flags) and not params_already_in_url:
             url += "?"
+        else:
```
Comment (Spacetown) [inline atlassian/rest_client.py]: This should be:
``python
elsif params_already_in_url:
```
At the moment the `&`i s added even if tere are no `parmas`and no `flags`.

**Review by Spacetown — CHANGES_REQUESTED** [review]

File: atlassian/rest_client.py (hunk 1 above)
Comment (Spacetown) [reply atlassian/rest_client.py]: Ok. Instead of the branch I selected the first commit.

**Comment by dim0x69** [comment]
Hi @Spacetown ,

Thanks for your suggestions! On first sight I (logically) don't see a difference in our codes, but yours is nicer to read :) 
I updated the Pull Request with  0fcc4d7 to include your idea.

**Comment by Spacetown** [comment]
> Hi @Spacetown ,
> 
> Thanks for your suggestions! On first sight I (logically) don't see a difference in our codes, but yours is nicer to read :)
> I updated the Pull Request with [0fcc4d7](https://github.com/atlassian-api/atlassian-python-api/commit/0fcc4d7ee78fa907be5835015da9f87b2547dd30) to include your idea.

I've checked again and you're right, it was ok. But I think, now it's easier to read.

**Comment by dim0x69** [comment]
I updated the PR with black formatting @Spacetown

**Comment by gonchik** [comment]
Thank you both for the improvement!

**Pull request merged**